src/components/string_input.tsx

```diff
@@ -32,19 +33,10 @@ const StringInput: FC<StringInputProps> = ({
     min,
     max,
     regexp,
+    hint,
 }) => {
     const [error, setError] = useState<string>('')
 
-    // use the cursor postion when user edits the data in the component
-    const [cursor, setCursor] = useState<number | null>(null)
-    const ref = useRef<HTMLInputElement>(null)
-
-    // Refresh after the first render and every time the component updates
-    useEffect(() => {
-        const input = ref.current
-        if (input) input.setSelectionRange(cursor, cursor)
-    }, [ref, cursor, value])
-
     const handleChange = (inputValue: string) => {
         if (typeof inputValue !== 'string') {
```

Comment:
Is it possible for inputValue to NOT be a string?

Reply:
Removed the typeof check for string